### PR TITLE
[Resolves #338] fix docs on viewing docs

### LINKED ISSUE #338: Fix documentation on how to view docs
Documentation on how to view the docs locally is out of date and requires updating.

### PR DESCRIPTION
**Pull request by JuanCanham**
**Replace the text above the line with a summary of the changes in your PR.
The more detailed you are, the better. Before submitting the PR, make sure the following are checked.**

-----------------

* [x] Wrote [good commit messages][1].
* [N/A] [Squashed related commits together][2] after the changes have been approved.
* [N/A] Commit message starts with `[Resolve #issue-number]` (if a related issue exists).
* [N/A] Added unit tests.
* [N/A ] Added integration tests (if applicable).
* [N/A] All unit tests (`make test`) are passing.
* [N/A] Used the same coding conventions as the rest of the project.
* [N/A] The new code doesn't generate flake8 (`make lint`) offenses.
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.

[1]: http://chris.beams.io/posts/git-commit/
[2]: https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -24,10 +24,10 @@ The docs can be built with:
 make docs-latest
 ```
 
-To serve the docs and watch for changes, run:
+To makes and serve the docs and watch for changes, run:
 
 ```shell
-make serve-latest
+make serve-docs-latest
```
Comment (theseanything) [inline docs/README.md]: This relates to the Makefile in parent dir. There is another Makefile under /docs which this is referring to.

But should probably add line to state that `cd /docs` and still add command for parent Makefile.

File: docs/README.md [hunk 2]
```diff
@@ -24,10 +24,10 @@ The docs can be built with:
 make docs-latest
 ```
 
-To serve the docs and watch for changes, run:
+To makes and serve the docs and watch for changes, run:
```
Comment (theseanything) [inline docs/README.md]: make*

Comment (JuanCanham) [reply docs/README.md]: Fixed

File: docs/README.md (hunk 1 above)
Comment (JuanCanham) [reply docs/README.md]: Added a help to the docs makefile (call it from normal makefile)

**Comment by ngfgrant** [comment]
@JuanCanham I'd like to merge this can you create a short issue for it and amend the commit message to include [Resolves xx] - thanks!

**Pull request merged**